server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -583,6 +591,18 @@ public void upgradeFileDeletes(ServerContext context, Ample.DataLevel level) {
     }
   }
 
+  public void removeAllScanServerRefs(ServerContext context, Ample.DataLevel level) {
```

Comment:
I think this upgrade code will only run if going from 2.0.x/1.10.x to 2.1.x.  Scan servers did not exist in 2.0.x/1.10.x, so this code may never do anything.

Reply:
Pulling this out to move to 3.1